web/modules/custom/dpl_patron_reg/src/Controller/DplPatronRegController.php

```diff
@@ -75,25 +79,22 @@ public static function create(ContainerInterface $container): DplPatronRegContro
   public function authRedirect(Request $request, string $client_name): TrustedRedirectResponse {
     $this->session->saveDestination();
 
-    $configuration = $this->config('openid_connect.settings.' . $client_name)
-      ->get('settings');
-    /** @var \Drupal\openid_connect\Plugin\OpenIDConnectClientInterface $client */
-    $client = $this->pluginManager->createInstance(
-      $client_name,
-      $configuration
-    );
```

Comment:
Please consider if we can drop the injected plugin manager with this being removed?